docs/details/blocks/GUI/Window.md

```diff
@@ -0,0 +1,43 @@
+A `GUI.Window` is a graphical user-interface window that can be used to display contents to, as well as to recieve inputs from, the user. Multiple `GUI.Window`s can be rendered within the `GFX.MainWindow` container.
+
+### `:Title` Text ###
+
+The `:Title` parameter value displays as text on the title-bar of the rendered window and hence can identify the window. However, for this to work, the title-bar display must not be suppressed (i.e. the flag `GuiWindowFlags.NoTitleBar` should not be passed to the `:Flag` parameter).
+
+### `:Pos`, `:Height` & `:Width` ###
```

Comment:
I would avoid using the `&` symbol and write `and` fully, as it has other meanings in programming in general.

Reply:
Agreed. Fixed here in other applicable *.md files too.